### PR TITLE
Add github actions to Artist

### LINKED ISSUE #41: Setup GitHub actions
https://docs.github.com/en/actions

### PR DESCRIPTION
**Pull request by sanchezz93**
Adding Github actions 
Resolves #41

### CODE REVIEW

File: .github/ci.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    # Install Gradle Deps
+    run: ./gradlew help
+  - name: Build project
+    run: ./gradlew assemble --stacktrace
+  - name: Run tests
+    run: ./gradlew test --stacktrace
+  - name: Upload snapshot (main only)
+    run: ./gradlew uploadArchives -PSONATYPE_NEXUS_USERNAME="$SONATYPE_NEXUS_USERNAME" -PSONATYPE_NEXUS_PASSWORD="$SONATYPE_NEXUS_PASSWORD"
+    env:
+      SONATYPE_NEXUS_USERNAME: ${{ secrets.SonatypeUsername }}
```
Comment (jbarr21) [inline .github/ci.yml:33]: are these secrets already setup?

File: .github/ci.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    uses: malinskiy/action-android/install-sdk@release/0.1.1
+  - name: Gradle Wrapper Validation
+    uses: gradle/wrapper-validation-action@v1
+  - name: Configure Gradle
+    # Install Gradle Deps
+    run: ./gradlew help
+  - name: Build project
+    run: ./gradlew assemble --stacktrace
+  - name: Run tests
+    run: ./gradlew test --stacktrace
```
Comment (jbarr21) [inline .github/ci.yml:29]: i think we still want do use `check`, too, right? (for checkstyle & lint)

File: .github/ci.yml (hunk 1 above)
Comment (sanchezz93) [reply .github/ci.yml:33]: Secrets have to be setup in Settings - https://docs.github.com/en/actions/reference/encrypted-secrets

Comment (jbarr21) [reply .github/ci.yml:33]: i added them. plz confirm the snapshot uploads properly after Actions run the first time

**Review by jbarr21 — APPROVED** [review]
remove the .travis.yml, otherwise lgtm

**Pull request merged**